snuba/util.py

```diff
@@ -55,21 +64,34 @@ def string_col(dataset, col):
         return 'toString({})'.format(escape_col(col))
 
 
-def escape_col(col):
-    if not col:
-        return col
-    elif SAFE_COL_RE.match(col):
-        # Column is safe to use without wrapping.
-        return col
+def escape_expression(expr: Optional[str], regex) -> Optional[str]:
+    if not expr:
+        return expr
```

Comment:
What's the case where this is `None`?

Reply:
I am not really sure, I just picked it up from the previous escape_col to be safe. This is not used only in query processing but in other places as well.

Reply:
Hrm, ok. I guess that makes sense to be safe but it'd be helpful to note that the necessity is unclear for this kind of thing.

Reply:
Ok, I will add a comment